Src/CommonProperties.props

```diff
@@ -7,30 +7,51 @@
     <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
 
     <CodeAnalysisCulture>en-US</CodeAnalysisCulture>
-    <CodeAnalysisRuleSet>$(SolutionDir)AutoFixture.ruleset</CodeAnalysisRuleSet>
+    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)\AutoFixture.ruleset</CodeAnalysisRuleSet>
+    <Features>IOperation</Features>
 
     <SignAssembly>True</SignAssembly>
-    <AssemblyOriginatorKeyFile>$(SolutionDir)AutoFixture.snk</AssemblyOriginatorKeyFile>
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)\AutoFixture.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
   <Choose>
+    <When Condition="$(TargetFramework.StartsWith('netstandard')) Or $(TargetFramework.StartsWith('netcoreapp'))">
+      <PropertyGroup>
+        <IsFullFramework>false</IsFullFramework>
+        <IsNetCore>true</IsNetCore>
+        <DefineConstants>$(DefineConstants);NETCORE</DefineConstants>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <PropertyGroup>
+        <IsFullFramework>true</IsFullFramework>
+        <IsNetCore>false</IsNetCore>
+        <DefineConstants>$(DefineConstants);NETFULL;SYSTEM_NET_MAIL;SYSTEM_RUNTIME_SERIALIZATION</DefineConstants>
```

Comment:
Good catch! I did that because it was simpler and there were a lot of standalone places that contained switches. But you are right - it's better to explicitly define which particular features we are switching on. I've split `NETFULL` to the more specific switchers.